ckanext/ontario_theme/templates/internal/scheming/package/resource_read.html

```diff
@@ -15,221 +15,233 @@
 {% set license = h.ontario_theme_get_license(pkg.license_id) %}
 
 {% block pre_primary %}
-  <section class="module module-resource" role="complementary">
-    {% if h.check_access('package_update', {'id':pkg.id }) %}
-      <header class="col-xs-12 page-header">
-      <div class="content_action">
-        {% link_for _('Edit'), named_route=pkg.type + '_resource.edit', id=pkg.name, resource_id=res.id, class_='btn btn-default content_action', icon='wrench' %}
-      </div>
-      </header>
-    {% endif %}
-    {% block resource_content %}
-      <div class="module-content">
-        <div class="col-lg-8 col-md-7 col-xs-12">
-          {% block resource_read_title %}
-            <h1 class="page-heading">{{ h.resource_display_name(res) }}</h1>
-          {% endblock %}
-          <div class="datafile-p" property="rdfs:label">
-            {% block resource_read_url %}
-              {% if res.url and h.is_url(res.url) %}
-                <p>{{ _('URL:') }} <a class="resource-url-analytics" href="{{ res.url }}" title="{{ res.url }}">{{ res.url }}</a></p>
-              {% elif res.url %}
-                <p>{{ _('URL:') }} {{ res.url }}</p>
-              {% endif %}
+  {# only display resource page content if open dataset or admin user #}
+  {% set is_admin = h.check_access('package_update', {'id':pkg.id }) %}
+  {% set is_open = (pkg.access_level == 'open') %}
+  {% if is_open or is_admin%}
+    <section class="module module-resource" role="complementary">
+      {% if h.check_access('package_update', {'id':pkg.id }) %}
```

Comment:
Do you need to check this again? `is_admin` and `is_open` are already satisfied

Reply:
the variable definition did not carry over to the different block, so had to redefine them.
I will use "Data Not Available" / "Données non disponibles"

Reply:
I meant, there is already an if condition on line 21, why do you need to check again on line 23?

Reply:
oh, right.
23 is from code already in place, and I was not sure whether to remove it or not. it is the only repeated-check happening. Removing it will simplify the logic, so i will make the change.